pkg/config/draftconfig.go

```diff
@@ -125,6 +127,61 @@ func (d *DraftConfig) ApplyDefaultVariables() error {
 	return nil
 }
 
+func (d *DraftConfig) ApplyDefaultVariablesForVersion(version string) error {
```

Comment:
nit - can you add a comment header like there is for ApplyDefaultVariables?

Also, more of a question than feedback since I imagine I'm missing something, but when would ApplyDefaultVariables be used and not this one?

Reply:
Eventually it can be removed once we migrate all templates!